tests/test_rxerr.cpp

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (c) 2016-2019 UAVCAN Team
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * Contributors: https://github.com/UAVCAN/libcanard/contributors
+ */
+
+#include <catch.hpp>
+#include <canard.h>
+
+//Independent implementation of ID layout from https://uavcan.org/Specification/4._CAN_bus_transport_layer/
```

Comment:
Am I correct that these defines need to be updated for v1.0 @pavel-kirienko?

Reply:
Yes.

Reply:
The defines are now updated, but tests will fail due to rest of Libcanard not updated yet.

Reply:
@aasmune Arriving late here, but what was the reason for including the ID/header changes in this "backport" PR? It seems to be that it would be better to drop the last PR, updating both header changes and tests in a PR of its own. Anything I'm missing?

Reply:
After talking to @aasmune I got the OK on removing the header update changes. We will fix this together with test update in a separate PR.